astroquery/ned/core.py

```diff
@@ -704,6 +704,23 @@ def _parse_result(self, response, verbose=False):
 Ned = NedClass()
 
 
+def _parse_radius(radius):
```

Comment:
@bsipocz I could only find two modules failing, `nvas` and `ned` , is there anything else that need to be edited before I submit another commit. I have locally fixed the issue for these two.